packages/eslint-plugin-example-typed-linting/README.md

```diff
@@ -13,7 +13,7 @@ import tseslint from 'typescript-eslint';
 export default tseslint.config(
     { ignores: ["lib"] },
     eslint.configs.recommended,
-    ...tseslint.configs.recommendedTypeChecked,
+    tseslint.configs.recommendedTypeChecked,
```

Comment:
In the latest versions, the `...` is no longer necessary. Arbitrary nesting is now allowed.